### ISSUE TITLE
Adding highest/lowest altitude

### ISSUE THREAD

**Issue by MTBehnke** [MTBehnke/NightSkyGuide#19]
Consider adding greatest and lowest altitude for each object

**Comment by TiiXel** [comment]
Is this part of https://github.com/MTBehnke/NightSkyGuide/commit/57195a9f1afbfe98120f03f438ea89d0c68ff6e0 ?

**Comment by MTBehnke** [comment]
Yes.  Still needs a little work.  Formatting, add lowest altitude for circumpolar, and ensure transit time is between rise/set or if circumpolar when to show last vs. next transit time.  Also hide extra rows for does not rise case.
